kitsune/sumo/static/sumo/js/analytics.js

```diff
@@ -54,12 +55,12 @@
   });
 })(jQuery);
 
-export default function trackEvent(category, action, label, value) {
+export default function trackEvent(name, parameters) {
+  console.log("--------------------");
+  console.log(`event: "${name}"`);
+  console.log(`parameters: ${JSON.stringify(parameters)}`);
+  console.log("--------------------");
```

Comment:
Is this intended?

Reply:
This is only temporary to help with some quick debugging, but I'll remove it before making this a real PR.